preset/rabbitmq/preset.go

```diff
@@ -121,6 +137,148 @@ func (p *P) setDefaults() {
 	}
 }
 
+func (p *P) loadFiles() error {
```

Comment:
Nitpick: I think that's it's easier to follow the code when methods are declared after they are used, so first you read the "main flow", and then look up any following implementation details used in it. I'm writing it just because I'm already submitting a review, but that's not a blocker in any way.